src/components/modern/LargeFiatTokenValueView.js

```diff
@@ -25,54 +25,72 @@ import { BigNumber } from 'bignumber.js';
 
 // Components
 import { Spacing } from 'components/modern/Layout';
-import Text from 'components/modern/Text';
+import TokenValueView from 'components/modern/TokenValueView';
+import FiatValueView from 'components/modern/FiatValueView';
 
 // Utils
-import { formatTokenValueWithoutSymbol } from 'utils/format';
 import { appFont, spacing } from 'utils/variables';
+import { useThemeColors } from 'utils/themes';
+import { getAssetValueInFiat } from 'utils/rates';
+
+// Selectors
+import { useFiatCurrency, useChainRates } from 'selectors';
 
 // Types
 import type { TextStyleProp } from 'utils/types/react-native';
-
+import type { Chain } from 'models/Chain';
 
 type Props = {|
   value: ?BigNumber,
+  assetAddress: string,
+  chain: Chain,
   symbol: string,
   style?: TextStyleProp,
 |};
 
 /**
  * Large (& stylized) component to display token value.
  */
-function LargeTokenValueView({ value, symbol, style }: Props) {
+function LargeFiatTokenValueView({ value, assetAddress, chain, symbol, style }: Props) {
+  const fiatCurrency = useFiatCurrency();
+  const colors = useThemeColors();
+  const rates = useChainRates(chain);
+  const balanceInFiat = getAssetValueInFiat(value, assetAddress, rates, fiatCurrency);
   if (!value) return null;
 
   return (
     <Container style={style}>
       {/* TokenValue & TokenSymbol are wrapped in plain RN Text in order to make baseline work */}
-      <RNText>
-        <TokenValue>{formatTokenValueWithoutSymbol(value, symbol, { stripTrailingZeros: true })}</TokenValue>
-        <Spacing w={spacing.extraSmall} />
-        <TokenSymbol>{symbol}</TokenSymbol>
+      <RNText style={styles.fiatTokenText} numberOfLines={1} adjustsFontSizeToFit>
+        <FiatValueView value={balanceInFiat} currency={fiatCurrency} variant="giant" style={styles.fiatValue} />
+        <Spacing w={spacing.small} />
+        <TokenValueView
+          value={value}
+          symbol={symbol}
+          variant="giant"
+          color={colors.secondaryText}
+          style={styles.tokenValue}
+        />
```

Comment:
Probably no need to use `variant` prop, since we're passing font-size through `style` prop

```suggestion
      <RNText style={styles.fiatTokenText} numberOfLines={1} adjustsFontSizeToFit>
        <FiatValueView value={balanceInFiat} currency={fiatCurrency} style={styles.fiatValue} />
        <Spacing w={spacing.small} />
        <TokenValueView
          value={value}
          symbol={symbol}
          color={colors.secondaryText}
          style={styles.tokenValue}
        />
```